R/clean_housing_stock_data.R

```diff
@@ -0,0 +1,27 @@
+#' Clean housing stock data
+#'
+#' There are several issues with the data frame
+#' read_housing_stock_data provides. First the column labels are not
+#' terribly great. Second, the periods column contains some sort of
+#' code instead of a year. Lastly, the regions column is superfluous,
+#' as there is only one region (the north of the Netherlands).
+#'
+#' @param df a data frame generated by read_housing_stock_data
+#' @return a data frame with four columns: Gemeentenaam, Provincienaam, year and `housing stock`.
+#' @import stringr
+#' @importFrom dplyr left_join
+#' @noRd
+
+clean_housing_stock_data <- function(df,codes_df) {
+  y <- filter(codes_df, Provincienaam == "Groningen" | Provincienaam == "Drenthe")
```

Comment:
Missing Friesland

Reply:
As discussed, it seems like we only need to look at Groningen and Drenthe, so just changing the var name is fine